### PR TITLE
Make constructor of Todo class const in examples

### PR DESCRIPTION
**Pull request by vandadnp**
Thank you for Riverpod

Since the Todo class is immutable pretty much, I made sure it's constructor is const

### CODE REVIEW

File: examples/todos/lib/todo.dart [hunk 1]
```diff
@@ -5,7 +5,7 @@ const _uuid = Uuid();
 
 /// A read-only description of a todo-item
 class Todo {
-  Todo({
+  const Todo({
```
Comment (rrousselGit) [inline examples/todos/lib/todo.dart:8]: Could you add `@immutable` then?

File: examples/todos/lib/main.dart [hunk 2]
```diff
@@ -15,7 +15,7 @@ final allFilterKey = UniqueKey();
 /// We are using [StateNotifierProvider] here as a `List<Todo>` is a complex
 /// object, with advanced business logic like how to edit a todo.
 final todoListProvider = StateNotifierProvider<TodoList, List<Todo>>((ref) {
-  return TodoList([
+  return TodoList(const [
```
Comment (rrousselGit) [inline examples/todos/lib/main.dart:18]:
```suggestion
  return const TodoList([
```

Comment (rrousselGit) [reply examples/todos/lib/main.dart:18]: Ignore that, got confused

**Review by rrousselGit — APPROVED** [review]
LGTM

File: examples/todos/lib/todo.dart (hunk 1 above)
Comment (vandadnp) [reply examples/todos/lib/todo.dart:8]: Yes sounds like a good idea, wonder why I didn't do that in the first place. Will fix soon...

**Comment by vandadnp** [comment]
@rrousselGit thanks for your review. I made the class immutable as per your suggestion. Ready when you are

**Pull request merged**